client/login/qr-code-login/index.jsx

```diff
@@ -0,0 +1,50 @@
+import { Card, Gridicon } from '@automattic/components';
+import classNames from 'classnames';
+import { localize } from 'i18n-calypso';
+import QRCode from 'qrcode.react';
+import qrCenter from 'calypso/assets/images/qr-login/wp.png';
+import GlobalNotices from 'calypso/components/global-notices';
+import Main from 'calypso/components/main';
+import { login } from 'calypso/lib/paths';
+
+import './style.scss';
+
+function QrCodeLogin( { translate, locale } ) {
+	const loginParameters = {
+		locale: locale,
```

Comment:
Nit: could be shortened to:

```suggestion
		locale,
```